boards/shields/nrf2240ek/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,7 @@
+# Options related to the nPM1300 regulator and the workaround
+# against voltage drops on the nRF2240 power supply pin VDDPALDO.
+CONFIG_SENSOR=y
+CONFIG_NPM1300_CHARGER=y
+# Ensure that FEM is initialized after the nPM1300 current limit on the nRF2240EK is set.
+CONFIG_SENSOR_INIT_PRIORITY=90
+CONFIG_MPSL_FEM_INIT_PRIORITY=91
```

Comment:
Since `DPPIC10` is used directly with the nrfx_dppi API, I suggest putting `CONFIG_NRFX_DPPI10=y` here. `CONFIG_NRFX_PPIB11=y` and `CONFIG_NRFX_PPIB21=y` as well.
`CONFIG_NRFX_DPPI0=y` should be set as well on nRF53.
Since GPPI is also used, `CONFIG_NRFX_GPPI` can be added as well.